crates/viewer/re_space_view_graph/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+authors.workspace = true
+description = "A space view that shows a graph (node-link diagram)."
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+name = "re_space_view_graph"
+publish = true
+readme = "README.md"
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
+
+[lints]
+workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+
+[dependencies]
+re_chunk.workspace = true
+re_format.workspace = true
+re_log.workspace = true
+re_log_types.workspace = true
+re_query.workspace = true
+re_renderer.workspace = true
+re_space_view.workspace = true
+re_tracing.workspace = true
+re_types.workspace = true
+re_ui.workspace = true
+re_viewer_context.workspace = true
+re_viewport_blueprint.workspace = true
+
+egui.workspace = true
+
+#TODO(grtlr): check if all of these are still needed
```

Comment:
FYI: cargo machete should do that at least on internal PR CI. (For this big PR, when we're close to done, I'll temporarily open a mirror PR so that we get the full CI.)